client/core/locale_ntfn.go

```diff
@@ -345,12 +345,12 @@ var ptBR = map[Topic]*translation{
 		subject:  "Erro ao Destravar Carteira",
 		template: "Conectado com carteira para completar o registro em %s, mas falha ao destrancar: %v",
 	},
-	// [ticker, error]
+	// [ticker, error], RETRANSLATE.
```

Comment:
Just a note to other reviewers.  This was my suggestion for keeping a reasonable although potentially outdated translation. https://github.com/decred/dcrdex/pull/1611/commits/fb983f188d305770368fc649391b3c6f65613eb1#r896098359